selfdrive/car/gm/carcontroller.py

```diff
@@ -25,13 +34,15 @@ def __init__(self, dbc_name, CP, VM):
     self.lka_icon_status_last = (False, False)
 
     self.params = CarControllerParams()
+    self.pitch = FirstOrderFilter(0., 0.09 * 4, DT_CTRL * 4) # runs at 25 Hz
 
     self.packer_pt = CANPacker(DBC[self.CP.carFingerprint]['pt'])
     self.packer_obj = CANPacker(DBC[self.CP.carFingerprint]['radar'])
     self.packer_ch = CANPacker(DBC[self.CP.carFingerprint]['chassis'])
 
   def update(self, CC, CS):
     actuators = CC.actuators
+    accel = actuators.accel
```

Comment:
use class variable so you don't need to set it when not an active control frame. check toyota/carcontroller